README.md

```diff
@@ -33,21 +33,90 @@ kustomizations:
   interval: 10m0s
 ```
 
+## Encrypt Kubernetes Secrets in a Git repository using Mozilla SOPS
+
+This chart is able to install a gpg secret key for usage with sops.
```

Comment:
```suggestion
Since your configuration will sometimes contain sensitive data, flux provides several mechanisms to store and handle that data securely.
This chart is able to install a gpg secret key for usage with sops. The public key from that same secret can then be used to encrypt the sensitive parts of your configuration, while flux will be able to decrypt it when pulling the configuration.
```